packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <p
+            :class="{
+                [$style['c-restaurantCard-iconText']]: true,
+                [$style['c-restaurantCard-iconText--bold']]: isBold
+            }"
+            aria-hidden="true"
```

Comment:
if this removes the entire component and it's children from the accessibility tree then I can't see how the accessible text stuff would get picked up by a screen reader. Perhaps I've misunderstood the purpose of this role but I'm unsure if we want it here. Can you let me know your thoughts please? :)

Reply:
this block of code is visually hidden, but it's picked up by the screen readers4
```
       <span
            class="is-visuallyHidden"
            data-test-id="icon-text-accessible-text">
            {{ accessibleText || text }}
        </span>
```